daffodil-io/src/test/scala/org/apache/daffodil/layers/TestBase64.scala

```diff
@@ -36,16 +36,25 @@ import org.apache.daffodil.exceptions.Assert
 class TestBase64 {
   Assert.usage(scala.util.Properties.isJavaAtLeast("1.8"))
 
-  val text = """Man is distinguished, not only by his reason, but by this singular passion from
-other animals, which is a lust of the mind, that by a perseverance of delight
-in the continued and indefatigable generation of knowledge, exceeds the short
-vehemence of any carnal pleasure.""".replace("\n", " ")
-
-  val b64Text = """TWFuIGlzIGRpc3Rpbmd1aXNoZWQsIG5vdCBvbmx5IGJ5IGhpcyByZWFzb24sIGJ1dCBieSB0aGlz
-IHNpbmd1bGFyIHBhc3Npb24gZnJvbSBvdGhlciBhbmltYWxzLCB3aGljaCBpcyBhIGx1c3Qgb2Yg
-dGhlIG1pbmQsIHRoYXQgYnkgYSBwZXJzZXZlcmFuY2Ugb2YgZGVsaWdodCBpbiB0aGUgY29udGlu
-dWVkIGFuZCBpbmRlZmF0aWdhYmxlIGdlbmVyYXRpb24gb2Yga25vd2xlZGdlLCBleGNlZWRzIHRo
-ZSBzaG9ydCB2ZWhlbWVuY2Ugb2YgYW55IGNhcm5hbCBwbGVhc3VyZS4=""".replace("\n", "\r\n")
+  //
+  // This is from Don Quixote which is entirely public domain
+  //
```

Comment:
Ok, well to simplify life I'll just change this text to the text of the Apache License....!

My only goal for this text is that it is readable, so that when debugging you can easily spot if it has been clobbered.

Reply:
I like it! That's definitely ASLv2 compatible.